crates/stackable-telemetry/CHANGELOG.md

```diff
@@ -7,13 +7,15 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Introduce common `Settings` and subscriber specific settings ([#901]).
+- Added support for logging to files ([#933]).
```

Comment:
suggestion: Use `Add` instead of `Added`.

```suggestion
- Add support for logging to files ([#933]).
```

note (non-blocking): I usually add newer entries in the changelog at the top. Feel free to move the entry or leave it as is.